.deepsource.toml

```diff
@@ -0,0 +1,12 @@
+version = 1
```

Comment:
please remove this.

If you think pydantic should run deepsource as part of CI:
* please create an issue to discuss it and and explain why it's a good idea (I don't personally think any of the things it has found here are particularly impressive)
* if agreed, we would then need to add setup deepsource to work with github statuses etc.

Reply:
> please remove this.
> 
> If you think pydantic should run deepsource as part of CI:
> 
>     * please create an issue to discuss it and and explain why it's a good idea (I don't personally think any of the things it has found here are particularly impressive)
> 
>     * if agreed, we would then need to add setup deepsource to work with github statuses etc.

Sure! Will open an issue on that.